app/authorities/qa/authorities/format.rb

```diff
@@ -24,8 +24,22 @@ def search(q)
 
     # Reformats the data received from the service
     def parse_authority_response(response)
-      [{ 'id' => response.first['@id'].to_s,
-         'label' => label.call(response) }]
+      id = find_response_id(response)
```

Comment:
I would prefer if QA Authorities used as similar as possible methods for finding the correct term.

This should work for every vocab that will only use URIs:
https://github.com/OregonDigital/OD2/blob/3b12af3604426842fa0d2c7ac93d885d050e34e3/app/authorities/qa/authorities/ethnographic_term.rb#L31-L33
Maybe it should be pushed up to Resource and overriden later.

We'll want a different strategy for label searches and maybe multiple vocabularies.

Reply:
We should develop a strategy for this for sure. But right now, I think its ok if we keep this separate per authority. Since each authority is its own, im not sure that each return will be similar. Id much rather rely on something that works for now rather than worry about refactoring. Once we get all the CVs in for single and multi, maybe we can take a look at all the different formats and find something that works for all, but thats probably 20 or 30 different endpoints.